distributed/utils_test.py

```diff
@@ -1122,41 +1106,6 @@ async def async_fn_outer():
     return _
 
 
-async def dump_cluster_state(
```

Comment:
Another option is to leave this functionality around but still disable it in `gen_cluster`. I'm not sure how much value there is to having `dump_cluster_state` around for diagnosing tricky deadlock situations.

Reply:
The functionality itself is part of Client. I'm not entirely sure what this utility function offers but I'd rather not maintain it.